protocol/v2/qbft/instance/observability.go

```diff
@@ -23,6 +24,16 @@ const (
 	commitStage   stage = "commit"
 )
 
+// roundChangeReason represents the reason for a round change in the QBFT protocol
+type roundChangeReason string
+
+const (
+	timeoutReason       roundChangeReason = "timeout"
+	partialQuorumReason roundChangeReason = "partial-quorum"
+	receivedReason      roundChangeReason = "received"
+	justifiedReason     roundChangeReason = "justified"
```

Comment:
IMO starting with `reason` would be more idiomatic (e.g. https://gobyexample.com/enums). If you decide to change it, perhaps we could also change the `stage` type

```suggestion
	reasonTimeout            roundChangeReason = "timeout"
	reasonPartialQuorum roundChangeReason = "partial-quorum"
	reasonReceived          roundChangeReason = "received"
	reasonJustified           roundChangeReason = "justified"
```

Reply:
yes, this is more consistent with go, honestly, I was used to using reason as a suffix, but I changed it, thanks!
`stage` is fixed as well